src/PrestaShopBundle/Controller/Admin/Sell/Order/InvoicesController.php

```diff
@@ -46,21 +51,21 @@ class InvoicesController extends FrameworkBundleAdminController
      * @return Response Template parameters
      */
     #[AdminSecurity("is_granted('read', request.get('_legacy_controller'))", message: 'Access denied.')]
-    public function indexAction(Request $request): Response
-    {
+    public function indexAction(
+        Request $request,
+        #[Autowire(service: 'prestashop.admin.order.invoices.by_date.form_handler')] Handler $byDateForm,
```

Comment:
Can you not use the `FormHandlerInterface` instead here? (and same in other methods)